src/components/navBar/demoBasicUsage/index.html

```diff
@@ -0,0 +1,11 @@
+<div ng-controller="AppCtrl" ng-cloak>
+  <md-content class="md-padding">
+    <md-nav-bar selected-tab-name="currentLink">
+      <md-nav-link nav-click="goto('page1')" link-name="page1">Page One</md-nav-link>
```

Comment:
It is an md-button so it renders as an anchor, or a button if you use a click event